src/SFA.DAS.EmployerIncentives.Api/Controllers/ApplicationController.cs

```diff
@@ -64,5 +65,18 @@ public async Task<IActionResult> GetApplication(long accountId, Guid application
 
             return Ok(response);
         }
+
+        [HttpGet]
+        [Route("/accounts/{accountId}/applications/{applicationId}/accountlegalentity")]
+        public async Task<IActionResult> GetApplicationLegalEntity(long accountId, Guid applicationId)
+        {
+            var result = await _mediator.Send(new GetApplicationLegalEntityQuery
+            {
+                AccountId = accountId,
+                ApplicationId = applicationId
+            });
+
+            return Ok(result);
+        }
```

Comment:
A couple of concerns here result appears to be an int which we pass back (not inside an object). But my main concern is since Mike has added the LegalEntityId to the GetApplication call, we could simply call the 'GetApplicationQuery' and pick up the value from there. However if we extend the GetApplication controller result to include the LegalEntityId then this endpoint is not needed at all.